crates/admin/src/schema_registry/updater.rs

```diff
@@ -260,6 +257,169 @@ impl SchemaUpdater {
         Ok(deployment_id)
     }
 
+    pub fn update_deployment(
+        &mut self,
+        deployment_id: DeploymentId,
+        deployment_metadata: DeploymentMetadata,
+        services: Vec<endpoint_manifest::Service>,
+    ) -> Result<(), SchemaError> {
```

Comment:
Maybe it's me, but I couldn't find a check that verifies the supported service protocol range is the same. This is definitely the most important thing to check, as there is a super high change that if they don't match, the invocation will get broken/corrupted.